physics/MP/GFDL/v3_2022/gfdl_cloud_microphys_v3_mod.F90

```diff
@@ -0,0 +1,7377 @@
+       regmax, fs2g_fac, fi2s_fac, fi2g_fac, do_sedi_melt, radr_flag, rads_flag,          &
+       radg_flag, do_wbf, do_psd_water_fall, do_psd_ice_fall, n0w_sig, n0i_sig, n0r_sig,  &
+       n0s_sig, n0g_sig, n0h_sig, n0w_exp, n0i_exp, n0r_exp, n0s_exp, n0g_exp, n0h_exp,   &
+       muw, mui, mur, mus, mug, muh, alinw, alini, alinr, alins, aling, alinh, blinw,     &
+       blini, blinr, blins, bling, blinh, do_new_acc_water, do_new_acc_ice, is_fac,       &
+       ss_fac, gs_fac, rh_fac_evap, rh_fac_cond, snow_grauple_combine, do_psd_water_num,  &
+       do_psd_ice_num, vdiffflag, rewfac, reifac, cp_heating, nconds, do_evap_timescale,  &
+       delay_cond_evap, do_subgrid_proc, fast_fr_mlt, fast_dep_sub, qi_gen
+    implicit none
+
+    private
+
+    ! -----------------------------------------------------------------------
+    ! interface functions
+    ! -----------------------------------------------------------------------
+
+    interface wqs
+        procedure wes_t
+        procedure wqs_trho
+        procedure wqs_ptqv
+    end interface wqs
+
+    interface mqs
+        procedure mes_t
+        procedure mqs_trho
+        procedure mqs_ptqv
+    end interface mqs
+
+    interface iqs
+        procedure ies_t
+        procedure iqs_trho
+        procedure iqs_ptqv
+    end interface iqs
+
+    interface mhc
+        procedure mhc3
+        procedure mhc4
+        procedure mhc6
+    end interface mhc
+
+    interface wet_bulb
+        procedure wet_bulb_dry
+        procedure wet_bulb_moist
+    end interface wet_bulb
+
+    ! -----------------------------------------------------------------------
+    ! public subroutines, functions, and variables
+    ! -----------------------------------------------------------------------
+
+    public :: gfdl_cloud_microphys_v3_mod_init
+    public :: gfdl_cloud_microphys_v3_mod_driver
+    public :: gfdl_cloud_microphys_v3_mod_end
+    public :: cld_sat_adj, cld_eff_rad, rad_ref
+    public :: qs_init, wqs, mqs, mqs3d
+    public :: c_liq, c_ice, rhow, wet_bulb
+    public :: cv_air, cv_vap, mtetw
+    public :: hlv, hlf, tice
+
+    ! -----------------------------------------------------------------------
+    ! precision definition
+    ! -----------------------------------------------------------------------
+
+    integer, parameter :: r8 = 8 ! double precision
+
+    ! -----------------------------------------------------------------------
+    ! initialization conditions
+    ! -----------------------------------------------------------------------
+
+    logical :: tables_are_initialized = .false. ! initialize satuation tables
+
+    ! -----------------------------------------------------------------------
+    ! physics constants
+    ! -----------------------------------------------------------------------
+
+    real, parameter :: grav = 9.80665 ! acceleration due to gravity (m/s^2), ref: IFS
```

Comment:
Yikes. Lots of redefining constants here. Why does GFDL MP need to have a different gravity or pi than the host or rest of physics?